VERSION

```diff
@@ -1 +1 @@
-1.0.0
+1.0.1
```

Comment:
Should we maybe create an issue for a simple shell script which adjust all the versions automatically so that we have to touch only one place like this file?

Reply:
A `releaseme.sh` would be nice, but the PR would still have all the files changed. The script could be interactive though and also be used for branching and tagging

Reply:
The problem I have as a reviewer with this version changes in a PR is, that I do not know if all version number are correctly adapted since I do not know which file contains a version number and has therefore to be changed.

The only thing I could do is to grep for `1.0.0` and look if everything is now `1.0.1` but if someone missed a number which is still `0.17.0` for instance I would miss it too.

When the `releaseme.sh` script would adjust all files I would have more confidence as a reviewer to accept it.